src/components/20-molecules/top-content-bar/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.2
+
+- Fix: Remove unnecessary code. (#2258)
```

Comment:
As explained to Ben before, the hash-initial numbers are github issue numbers, not pull-requests. You could open a n issue that tersely explains what code needs to be removed, and mention its number here.

Reply:
@markus-walther nothing was ever explained to me except that you mentioned that you had to fix something.
Please document how you want this to be done. So you say Bozo needs to open an issue even if we have a WEBHUB Jira item?

Reply:
I also thought that this is a pull request number. I removed the "(#2258)" part from the changelog.